app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,45 @@
+class ApiController
+  module Chargeback
+    # Request
+    # {
+    #   "action":"assign",
+    #
+    #   "category": "tenant",
+    #   "c_id": 1
+    # }
+    def chargebacks_assign_resource(object, _type, _id, data)
+      klass = Object.const_get(data['category'].camelcase)
```

Comment:
I'm worried about "category" being potentially confusing here since we already have a domain concept `Category`